README.md

```diff
@@ -1,10 +1,17 @@
 # LSP-css
-CSS, SCSS, LESS support for Sublime's LSP plugin.
 
-* Install [LSP](https://packagecontrol.io/packages/LSP) and `LSP-css` from Package Control. If you use Sass install [Sass](https://packagecontrol.io/packages/Sass).
-If you use Less install [LESS](https://packagecontrol.io/packages/LESS).
+CSS, SCSS, LESS support for Sublime's LSP plugin provided through [vscode-css-languageserver](https://github.com/vscode-langservers/vscode-css-languageserver).
+
+### Installation
+
+* Install [LSP](https://packagecontrol.io/packages/LSP) and `LSP-css` from Package Control.
+  If you use Sass install [Sass](https://packagecontrol.io/packages/Sass).
+  If you use Less install [LESS](https://packagecontrol.io/packages/LESS).
 * Restart Sublime.
 
 ### Configuration
 
-Configure the css language server by accessing `Preferences > Package Settings > LSP > Servers > LSP-css`.
\ No newline at end of file
+There are some ways to configure the language server.
```

Comment:
Technically we are configuring package and server, not just server.

Reply:
How about

```diff
- There are some ways to configure the language server.
+ There are some ways to configure the package and the language server.
```

Reply:
Sure.